UPGRADE-4.2.md

```diff
@@ -0,0 +1,24 @@
+UPGRADE FROM 4.1 to 4.2
+=======================
+
+Form
+----
+
+ * Deprecated calling `FormRenderer::searchAndRenderBlock` for fields which were already rendered. 
+ Instead of expecting such calls return empty string, check if field has already been rendered.
```

Comment:
The whole block must be indented properly under the `* ` to fix rendering.

Reply:
[...] calls to return [...] if the field [...]